src/cplscheme/MultiCouplingScheme.cpp

```diff
@@ -149,9 +146,6 @@ void MultiCouplingScheme::addDataToSend(
   PtrCouplingData     ptrCplData(new CouplingData(data, std::move(mesh), initialize, getExtrapolationOrder()));
   DataMap::value_type dataPair = std::make_pair(id, ptrCplData);
   _sendDataVector[to].insert(dataPair);
```

Comment:
You could shorten this even more by directly calling `emplace(id, ptrCplData)`.

Same below

Reply:
Done [21254c4](https://github.com/precice/precice/pull/1185/commits/21254c426aca3b4fd23c81d546f56b4cc7477843).